docs/guide/adapters/flashing/flashing_via_cc2538-bsl.md

```diff
@@ -0,0 +1,73 @@
+---
+---
+# Flashing the firmware via cc2538-bsl
+
+[TI CC13xx/CC2538/CC26xx Serial Boot Loader](https://github.com/JelmerT/cc2538-bsl)  
+
+CC2538-bsl is a python script by [Jelmer Tiete](https://github.com/JelmerT) that communicates with the boot loader of the Texas Instruments CC2538, CC26xx and CC13xx SoCs (System on Chips).  
+
+In this case we use a Sonoff Zigbee 3.0 USB Dongle Plus and flash the firmware without opening the case to press the "Boot" button.
+
+## Prepare your system (debian based)
+
+1. Install python and python-pip.  
+```bash 
+sudo apt update && sudo apt install python3 python3-pip 
+```
+2. Install needed python packages.  
+```bash
+sudo pip3 install pyserial intelhex
+```
+## Download cc2538-bsl
+3. We need the ITead_Sonoff_Zigbee-delay "bootloader" feature.  
+```bash
+mkdir z2m_fw-upgrade
+cd z2m_fw-upgrade
```

Comment:
Doesn't seem to be necessary, the unzip wil already create a separate directory.

Reply:
Yes, You're right, but I wanted everything in one directory to keep things tidy.
If You want, i can give a hint, for my reasons.

Reply:
I will rename the directory to cc2538-bsl instead (makes more sense then `z2m_fw-upgrade`)